csharp/src/Drivers/Interop/Snowflake/Apache.Arrow.Adbc.Drivers.Interop.Snowflake.csproj

```diff
@@ -26,13 +36,67 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
-  <ItemGroup>
+
+  <!-- What runs in the pipeline. Assumes the Go builds have happened. -->
+  <ItemGroup Condition="$([MSBuild]::IsOSPlatform('Linux'))">
     <Content Include="readme.md">
       <Pack>true</Pack>
       <PackagePath>\</PackagePath>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-  </ItemGroup>
+
+    <!-- Windows/DLLs -->
+    <Content Include="libadbc_driver_snowflake.dll">
+      <Pack>true</Pack>
+      <PackagePath>lib\netstandard2.0</PackagePath>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="libadbc_driver_snowflake.dll">
+      <Pack>true</Pack>
+      <PackagePath>lib\net472</PackagePath>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="libadbc_driver_snowflake.dll">
+      <Pack>true</Pack>
+      <PackagePath>lib\net6.0</PackagePath>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+
+    <!-- Linux/so -->
+    <Content Include="libadbc_driver_snowflake.so">
+      <Pack>true</Pack>
+      <PackagePath>lib\netstandard2.0</PackagePath>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="libadbc_driver_snowflake.so">
+      <Pack>true</Pack>
+      <PackagePath>lib\net472</PackagePath>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="libadbc_driver_snowflake.so">
+      <Pack>true</Pack>
+      <PackagePath>lib\net6.0</PackagePath>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+
+    <!-- Mac/dylib -->
+    <Content Include="libadbc_driver_snowflake.dylib">
```

Comment:
Are Go-built binaries "fat" or would we need separate binaries for x64 and arm64?

Reply:
You would need separate binaries for x64 and arm64. You can control the build by using `GOARCH=arm64` or `GOARCH=amd64` if you want to cross compile